packages/common-all/src/store/INoteStore.ts

```diff
@@ -22,6 +22,15 @@ export interface INoteStore<K> {
    */
   get(key: K): Promise<RespV3<NoteProps>>;
 
+  /**
+   * Bulk get NoteProps by list of key
+   * If no notes are found, return empty list.
+   *
+   * @param key: keys of NoteProps
+   * @return list of NoteProps
+   */
+  bulkGet(keys: K[]): Promise<RespV3<NoteProps>[]>;
```

Comment:
should this be `<RespV3<NoteProps[]>>` instead of `<RespV3<NoteProps>[]>`?

that is - do we want to report errors on the overall API call or on a per-get basis.  I think the former makes more sense.